src/detect.c

```diff
@@ -198,6 +198,49 @@ static void DetectRun(ThreadVars *th_v,
     SCReturn;
 }
 
+/** \internal
+ */
+static void DetectRunPacketHook(ThreadVars *th_v, const DetectEngineCtx *de_ctx,
```

Comment:
nit: Should this have "Pre" in the name? Seems to only be used by pre hooks..

Reply:
I don't think so, there is nothing inherently "Pre" about that func. Could be reused for other hooks.